src/utils/bitreader.cpp

```diff
@@ -1,7 +1,7 @@
 #include "bitreader.h"
 #include <stdio.h>
```

Comment:
`stdio.h` er deprecated i C++, brug `cstdio` i stedet for.

Reply:
fixed